plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts

```diff
@@ -584,18 +578,19 @@ describe('ingester', () => {
 
             // Call the second ingester to receive the messages. The revocation should still be in progress meaning they are "paused" for a bit
             // Once the revocation is complete the second ingester should receive the messages but drop most of them as they got flushes by the revoke
+            mockAssignments.mockImplementation(() => [createTP(2), createTP(3)])
             await otherIngester.handleEachBatch([...partitionMsgs2, createMessage('session_id_4', 2)])
             await Promise.all(rebalancePromises)
 
             // Should still have the partition 1 sessions that didnt move
             expect(
                 Object.values(ingester.sessions).map((x) => `${x.partition}:${x.sessionId}:${x.buffer.count}`)
             ).toEqual(['1:session_id_1:1', '1:session_id_2:1'])
-
             // Should have session_id_4 but not session_id_3 as it was flushed
+
             expect(
                 Object.values(otherIngester.sessions).map((x) => `${x.partition}:${x.sessionId}:${x.buffer.count}`)
-            ).toEqual(['2:session_id_4:1'])
+            ).toEqual(['2:session_id_3:1', '2:session_id_4:2'])
```

Comment:
This was testing the partition locking that I got rid of for now, hence why there is more sessions than before